### DISCUSSION TITLE
Rename createStore().res to restore

### DISCUSSION THREAD

**Discussion by FlatMapIO** [pmndrs/jotai#1646] [General]
I looked through the source code to understand what `store.res` really is, and I suggest clarifying the naming of this api

**Comment by dai-shi** [comment]
Thanks for the suggestion. Yeah, we can consider the naming. Though, I'm not sure even if restore is correct, or how it should be used. Meanwhile, we should add unstable prefix. Appreciate for the feedback.

**Reply by dai-shi** [reply]
Made it dev only: #1648
